src/ResourceManager/Sql/Commands.Sql/AdvancedThreatProtection/Model/ManagedInstanceAdvancedThreatProtectionPolicyModel.cs

```diff
@@ -0,0 +1,37 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.WindowsAzure.Commands.Common.Attributes;
+
+namespace Microsoft.Azure.Commands.Sql.AdvancedThreatProtection.Model
+{
+    public class ManagedInstanceAdvancedThreatProtectionPolicyModel
+    {
+        /// <summary>
+        /// Gets or sets the resource group
+        /// </summary>
+        public string ResourceGroupName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the managed instance name
+        /// </summary>
+        public string ManagedInstanceName { get; set; }
```

Comment:
InstanceName

Reply:
In AzureSqlManagedInstanceModel, the managed instance name property is also called ManagedInstanceName. Do you still want me to rename to InstanceName?